packages/pharos/src/components/image-card/pharos-image-card.ts

```diff
@@ -125,6 +133,7 @@ export class PharosImageCard extends LitElement {
     return html`<pharos-link
       class="card__link--collection"
       href="${this.link}"
+      aria-label=${ifDefined(this.imageLinkLabel)}
```

Comment:
I went with an `aria-label` here because thats what we use in the standard pharos link, but maybe i should just use `label` and let the `pharos-link` do more work. I can try that out via yalc in a moment.